### PR TITLE
fix release ci

### LINKED ISSUE #1766: [Feature] maven仓库还未收录0.12.0版本
### Feature Description (功能描述)

https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-core
还没有最新的0.12.0版本
可以帮忙上传收录下么

### PR DESCRIPTION
**Pull request by coderzc**
fix #1766

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -31,7 +31,10 @@ jobs:
       - name: Package
         run: |
           mvn clean package -DskipTests
-          mv "$(find hugegraph-*.tar.gz | head -1)" hugegraph-${TAG_NAME/#v/}.tar.gz
+
+      - name: Rename file
```
Comment (javeme) [inline .github/workflows/release.yml:35]: why need to split into 2 stages? can put "mvn clean package -DskipTests" into "Rename file" stage?

Comment (coderzc) [reply .github/workflows/release.yml:35]: because of the "Rename file" stage is not unessential

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -31,7 +31,10 @@ jobs:
...
+        if: contains(env.TAG_NAME, "-")
```
Comment (javeme) [inline .github/workflows/release.yml:36]: fixed error `mv: '***-0.12.0.tar.gz' and '***-0.12.0.tar.gz' are the same file`

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -31,7 +31,10 @@ jobs:
...
+        run: mv "$(find hugegraph-*.tar.gz | head -1)" hugegraph-${TAG_NAME/#v/}.tar.gz
 
       - uses: xresloader/upload-to-github-release@master
```
Comment (javeme) [inline .github/workflows/release.yml:39]: also add name "Upload to GitHub"

File: .github/workflows/release.yml (hunk 1 above)
Comment (javeme) [reply .github/workflows/release.yml:35]: OK

File: .github/workflows/release.yml (hunk 2 above)
Comment (javeme) [inline .github/workflows/release.yml:36]: only need to rename when TAG_NAME is like 0.12.0-b2?

Comment (coderzc) [reply .github/workflows/release.yml:36]: yes

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]
could sign new CLA again

**Comment by coderzc** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by javeme — APPROVED** [review]

**Pull request merged**